model.js

```diff
@@ -1,13 +1,53 @@
 const connection = require('/home/veronika/northcoders/backend/be-nc-games/db/connection.js');
+const fs = require('fs');
 
 exports.selectCategory=()=>{
     
-   
-       // return Promise.reject({ status: 400, msg: "Invalid sort query"})
-    
 
     return connection.query(`SELECT * FROM categories;`)
     .then((result)=>{
         return result.rows;
     })
 }
+
+
+
+exports.getFile = () => {
+  return new Promise((resolve, reject) => {
+    fs.readFile('endpoints.json', 'utf8', (err, data) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(JSON.parse(data));
+      }
+    });
+  });
+};
```

Comment:
As you're not making any queries to the database it's not necessary to have anything in the model for this endpoint. You can do all this logic in the controller 🙂